For Advanced/020.md

```diff
@@ -0,0 +1,42 @@
+# Elementary ID
+
+Many of the `selenium.` commands allow to select an element on a page in order to click it, get or set its attribute, or perform other actions. They use the `search` and the `by` arguments to pinpoint the required element: you provide the phrase to search for an element after the `search` argument and the element selector after the `by` argument.
```

Comment:
Many of ... allow to select to click / get / set / perform other actions - IMHO this is what all Selenium commands do, especially with the "other actions" part ;-) => consider removal "Many of " or remove "or perform other actions".

Reply:
```suggestion
The `selenium.` commands allow to select an element on a page in order to click it, get or set its attribute, or perform other actions. They use the `search` and the `by` arguments to pinpoint the required element: you provide the phrase to search for an element after the `search` argument and the element selector after the `by` argument.
```